templates/log-book-report/pdf_log_book.go

```diff
@@ -0,0 +1,225 @@
+/*
+ * This example showcases the usage of creator templates by creating a sample
+ * log book report.
+ *
+ * Run as: go pdf_log_book.go
+ */
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io.
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+}
+
+// Item represents a log book item.
+type Item struct {
+	Source        string `json:"Source"`
+	Manufacturer  string `json:"Manufacturer"`
+	Model         string `json:"Model"`
+	VIN           string `json:"VIN"`
+	Received      string `json:"Received"`
+	Sent          string `json:"Sent"`
+	Buyer_Name    string `json:"Buyer_Name"`
```

Comment:
Please don't use field names with underscores. `Buyer_Name` -> `BuyerName`. There are other fields in this struct like that.